tests/test-samba-container.sh

```diff
@@ -1,44 +1,71 @@
 #!/bin/bash
 
-LOCAL_TAG="${LOCAL_TAG:-samba-server:latest}"
 
+
+echo "determining container command."
 if [ -z "${CONTAINER_CMD}" ]; then
-	CONTAINER_CMD=$(command -v docker || echo "")
+	CONTAINER_CMD=$(command -v docker || echo )
 fi
 if [ -z "${CONTAINER_CMD}" ]; then
-	CONTAINER_CMD=$(command -v podman || echo "")
+	CONTAINER_CMD=$(command -v podman || echo )
 fi
-
+if [ -z "${CONTAINER_CMD}" ]; then
+echo "Error determining container command."
+exit 1
+fi
+echo "container command: '${CONTAINER_CMD}'."
+echo "creating temporary directory."
 TMPDIR="$(mktemp -d)"
 rc=$?
 
 if [ $rc -ne 0 ]; then
-	echo "Error creating temporary directory"
-else
-	CONTAINER_ID="$(${CONTAINER_CMD} run --network=none --name samba \
-		--volume="${TMPDIR}":/share:Z --rm  -d "${LOCAL_TAG}")"
-	rc=$?
+	echo "Error creating temporary directory."
+	exit 1
+fi
+echo "temporary directory: '${TMPDIR}'"
+echo "starting Samba container."
+CONTAINER_ID="$(${CONTAINER_CMD} run --network=none --name samba \
+	--volume="${TMPDIR}":/share:Z --rm  -d "${LOCAL_TAG}")"
+rc=$?
 
-	if [ $rc -ne 0 ]; then
-		echo "Error running samba container"
-	else
-		echo "Container started, ID: '${CONTAINER_ID}'"
+if [ $rc -ne 0 ]; then
+	echo "Error running samba container"
+	exit 1
+fi
+echo "Container started, ID: '${CONTAINER_ID}'"
 
-		# give samba a second to come up
-		sleep 1
+# give samba a second to come up
+sleep 1
 
-		${CONTAINER_CMD} exec "${CONTAINER_ID}" smbclient -U% -L 127.0.0.1
-		rc=$?
+echo "Listing samba shares"
+	${CONTAINER_CMD} exec "${CONTAINER_ID}" smbclient -U% -L 127.0.0.1
+	rc=$?
```

Comment:
```suggestion
${CONTAINER_CMD} exec "${CONTAINER_ID}" smbclient -U% -L 127.0.0.1
rc=$?
```

Reply:
strange - I thought I had fixed those. anyway: committed now. I'm going to squash these changes into an earlier commit

Reply:
> strange - I thought I had fixed those. anyway: committed now. I'm going to squash these changes into an earlier commit


done, by the way